src/clarity-angular/popover/signpost/signpost.ts

```diff
@@ -0,0 +1,347 @@
+
+    /**********
+     * @param _open
+     * @description
+     * A property used to hold the open/closed state of the popover content.
+     * @type {boolean}
+     * @private
+     */
+    private _open: boolean;
+
+    /*********
+     * @param openSubscription
+     *
+     * @description
+     * Used for resource allocation and clean up related to subscribing to the IfOpen service changes.
+     * @type {Subscription}
+     * @private
+     */
+    private openSubscription: Subscription;
+
+    /**********
+     * @param anchorPoint
+     *
+     * @description
+     * A Point that is used by the popover directive to determine where the popover should be anchored relative to the
+     * trigger icon. For example, Point.RIGHT_CENTER will anchor the popover template to the right center of the trigger
+     * icon.
+     *
+     * @default Point.RIGHT_CENTER
+     * @type {Point}
+     */
+    public anchorPoint: Point = Point.RIGHT_CENTER; // default if menuPosition isn't set
+
+    /**********
+     * @param popoverPoint
+     *
+     * @description
+     * A Point that is used by the popover directive to determine where the popover should be placed in relation to the
+     * trigger. For example, Point.LEFT_CENTER will position the rendered popover template on the LEFT_CENTER point of
+     * popover container relative to the anchorPoint.
+     *
+     * @default Point.LEFT_CENTER
+     * @type {Point}
+     */
+    public popoverPoint: Point = Point.LEFT_CENTER; // default if menuPosition isn't set
+
+    /********
+     * @param signpostOptions
+     *
+     * @description options that configure the popover used to display a SignpostContent component. We do not allow
+     * multiple signposts to be open at the same time. offsetY and offsetX are used to 'move' the popover contining the
+     * SignpostContent vertically (offsetY) or horizontally (offsetX) as needed for the Position. Both offsets take
+     * both posative and negative values.
+     *
+     * @type {{allowMultipleOpen: boolean; offsetY: number; offsetX: number}}
+     */
+    public signpostOptions: PopoverOptions = {
+        allowMultipleOpen: false,
+        offsetY: this._offsetY, //TODO: set these based on signPostPosition case
+        offsetX: this._offsetX
+    };
+
+    /**********
+     *
+     * @function open
+     *
+     * @description
+     * A setter for the _open value which is subscribed to the IfOpen service of the IfOpen structural directive that
+     * controls the creation and removal of the SignpostContent template.
+     *
+     * @param value<boolean>
+     */
+    public set open(value: boolean) {
+        this._open = value;
+    }
+
+
+    /*********
+     *
+     * @function open
+     *
+     * @description
+     * A getter function for the current value of the open state for the SignpostContent popover.
+     * @returns {boolean}
+     */
+    public get open(): boolean {
+        return this._open;
+    }
```

Comment:
Do you still need this? I should simply use `this.ifOpenService.open` directly, no need to mirror the property.